spec/models/health/qualifying_activity_spec.rb

```diff
@@ -11,7 +11,7 @@
       qualifying_activity.calculate_payability!
 
       expect(pre_enrollment_activity.naturally_payable).to be true
-      expect(pre_enrollment_activity.compute_valid_unpayable?).to be true
+      expect(pre_enrollment_activity.compute_valid_unpayable).to contain_exactly(:outside_enrollment, :outside_enrollment)
```

Comment:
Are we going to see duplicates?  Seems like we should call `uniq` on the result before returning from `compute_valid_unpayable` if we are.